src/pages/graphql/schema/checkout/queries/get-payment-sdk.md

```diff
@@ -0,0 +1,99 @@
+---
+title: getPaymentSDK query
+---
+
+# getPaymentSDK query
+
+The `getPaymentSDK` query gets the payment paypal SDK urls and other related values.
+
+## Syntax
+
+```graphql
+{ 
```

Comment:
This is the syntax
{
    getPaymentSDK(
        location: PaymentLocation! 
    ) GetPaymentSDKOutput
   }